tests/cabal.project

```diff
@@ -0,0 +1,9 @@
+packages: bytestring-tests.cabal
+          ../bytestring.cabal
+
+allow-newer: *:bytestring
+           , *:ghc-prim
+           , regex-posix:base
+
+package bytesting-tests
+    tests: True
```

Comment:
Why not

```suggestion
tests: True
```

?

Reply:
We don't want to test any of our dependencies.  I am not sure what the scope of that is, it may cover dependencies.

Reply:
The default scope are the packages listed above AFAIK.